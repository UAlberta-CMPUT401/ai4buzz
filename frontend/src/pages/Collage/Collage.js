import styles from './Collage.module.css';

const Collage = ({ imgString }) => {
  return (
    <div className={styles.collagePage}>
      <img
        className={styles.collage}
        src={`data:image/jpeg;base64,${imgString}`}
        alt="collage"
      />
    </div>
  );
};

export default Collage;
