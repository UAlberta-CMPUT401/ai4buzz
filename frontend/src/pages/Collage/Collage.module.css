.collagePage {
  padding-top: 3rem;
  display: flex;
  justify-content: center;
}

.collage {
  width: 80%;
  max-width: 80%;
}
