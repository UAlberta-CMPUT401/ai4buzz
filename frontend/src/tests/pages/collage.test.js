import React from 'react';
import renderer from 'react-test-renderer';
import Collage from '../../pages/Collage/Collage';

describe('Test Login Page', () => {
  it('Renders login page correctly', () => {
    const imgString =
      'iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==';
    const tree = renderer.create(<Collage imgString={imgString} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
