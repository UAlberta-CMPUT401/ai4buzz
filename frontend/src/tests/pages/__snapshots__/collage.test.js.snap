// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Login Page Renders login page correctly 1`] = `
<div
  className="collagePage"
>
  <img
    alt="collage"
    className="collage"
    src="data:image/jpeg;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg=="
  />
</div>
`;
